deployment/check-environment.sh

```diff
@@ -47,12 +47,28 @@ print("Attempting load of fortran-based modules... ", end='', flush=True)
 from wagl import singlefile_workflow
 print("✅")
 
-# The previous import of wagl should have initialised the filters.
+# TODO CHECK: The previous import of wagl should have initialised the filters.
 print("Attempting hdf5 blosc compression...", end='', flush=True)
 import h5py
+import hdf5plugin
 import tempfile
+
+# Test different forms of compression usage as per the hdf5plugin docs:
+# https://hdf5plugin.readthedocs.io/en/stable/usage.html
+# use "hdf5plugin.Blosc()" style instead of int code to prevent crashes
+print("Attempting hdf5 blosc compression...", end='', flush=True)
 f = h5py.File(tempfile.mktemp('-test.h5'),'w')
-dset = f.create_dataset("myData", (100, 100), compression=32001)
```

Comment:
I don't know who created this test, but it will never work, as blosc requires additional `compression_opts` to be passed through in the form of a tuple of 7 integers, eg `(0, 0, 0, 0, 9, 1, 5)`